client/src/components/teamPage/TeamPage.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from "react";
+import { Box } from "@mui/material";
+import "../Styles/MainPage.scss";
+import { SearchBar } from "../SearchBar";
+import Modal from "react-modal";
+import { Typography } from "@mui/material";
+
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemAvatar from "@mui/material/ListItemAvatar";
+import ListItemText from "@mui/material/ListItemText";
+import Divider from "@mui/material/Divider";
+import Button from "@mui/material/Button";
+import CloseIcon from "@mui/icons-material/Close";
+import PetyaPhoto from "src/images/Petya.jpg";
+import DimitrinaPhoto from "src/images/Dimitrina.jpg";
+import SimeonPhoto from "src/images/Simeon.jpg";
+import IvayloPhoto from "src/images/Ivaylo.jpg";
+import ZahariPhoto from "src/images/Zahari.jpg";
+import GeorgiPhoto from "src/images/Georgi.jpg";
+import PavelPhoto from "src/images/Pavel.jpg";
+import BozhidarPhoto from "src/images/Bozhidar.jpg";
+
+const team = [
+  {
+    name: "Petya Marinova",
+    avatar: PetyaPhoto,
+    personInfo: ["Petya Marinova"],
```

Comment:
I think we don't need this personalInfo property, because we already have the name of the developer. Otherwise you can set the information from About section from LinkedIn or ask the person. And same for other properties that include this.

Reply:
Personal info is actually more information for the modal

Reply:
The popup is in the same place. It is better when the name is visible.

Reply:
Fixed.